### DISCUSSION TITLE
Mail headers of incoming email pushed via HTTP endpoint

### DISCUSSION THREAD

**Discussion by globalexport** [postalserver/postal#3259] [Help with using Postal] [answered]
Hi! 

I would like to verify DKIM headers of incoming emails after they were routed to my own https endpoint but I cannot find any headers in the request body.

Has this not been implemented or is it possible to configure the inclusion of the original headers somewhere? 

Thank you!

**Comment by globalexport** [comment]
I did not try this:
https://docs.postalserver.io/developer/http-payloads#the-raw-message-payload

Will this include all headers?

**Reply by willpower232** [reply, marked as answer]
I think so but I'm not sure. Worth trying for sure.

**Reply by globalexport** [reply]
The raw message indeed provides the DKIM headers of the original sender.
